node-src/lib/log.ts

```diff
@@ -93,25 +115,36 @@ const fileLogger = {
   },
 };
 
-export const createLogger = () => {
-  let level = (LOG_LEVEL.toLowerCase() as keyof typeof LOG_LEVELS) || DEFAULT_LEVEL;
-  if (DISABLE_LOGGING === 'true') level = 'silent';
+/* eslint-disable-next-line complexity */
+export const createLogger = (flags: Flags, options?: Partial<Options>) => {
```

Comment:
Out of curiosity, why didn't we use a log package instead of writing it ourselves?

Reply:
Because I tried to have the least amount of dependencies, and I needed one that could queue up log messages to print them later (due to how interactive mode works, we can't print anything while Listr is running).